docs/IE_PLUGIN_DG/plugin_transformation_pipeline/low_precision_transformations/lpt_attributes.md

```diff
@@ -33,11 +33,11 @@
 >  - `Quantization alignment` - the attribute defines quantization granularity in runtime: per-channel or per-tensor quantization.
 >  - `Quantization granularity` - the attribute is set by plugin to define quantization granularity: per-channel or per-tensor quantization.
 >
-> `Required` attribute group defines if attribute usage is required to get an optimal model during transformation:
+> `Required` attribute group checks if attribute usage is required to get an optimal model during transformation:
```

Comment:
For me the "checks" word could be little confusing, as the 'Required' column specifies attribute usage for optimal model. So it does not check, but specifies the behavior.

Reply:
Fixed in https://github.com/openvinotoolkit/openvino/pull/14511/commits/3d4b038076c68a94245d0e2bd2dc29211cb37f8b